src/main/java/p2/dao/impl/InterestDAO.java

```diff
@@ -20,8 +19,8 @@
 		List<Interest> list = null;
 
 		try {
-			Query query = session.createQuery("FROM Interest WHERE product.id = :productId");
```

Comment:
This is working. What is the problem?

Reply:
the table's name is Interests, the column name is product_id and the placeholder should be the same case as the column name